testbeam_analysis/dut_alignment.py

```diff
@@ -858,35 +842,41 @@ def alignment(input_track_candidates_file, input_alignment_file, n_pixels, pixel
         alignment_parameters = _create_alignment_array(n_duts)
         alignment_parameters['translation_z'] = prealignment['z']
 
-        if initial_rotation:
-            if isinstance(initial_rotation[0], Iterable):
-                for dut_index in range(n_duts):
+        beta = np.arctan(prealignment[align_telescope[-1]]["column_c0"] / prealignment[align_telescope[-1]]["z"])
```

Comment:
Please document the code more. Here the start parameters are deduced from pre-alignment?